dcos_launch/gce.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+
+from dcos_launch import util
+from dcos_test_utils import gce
+from dcos_test_utils.helpers import Host
+
+from googleapiclient.errors import HttpError
+
+log = logging.getLogger(__name__)
+
+
+class BareClusterLauncher(util.AbstractLauncher):
+    # Launches a homogeneous cluster of plain GMIs intended for onprem DC/OS
+    def __init__(self, config: dict):
+        credentials_path = util.set_from_env('GOOGLE_APPLICATION_CREDENTIALS')
+        credentials = util.read_file(credentials_path)
```

Comment:
Are the file system dependency and environment variable documented somewhere for a first-time user? (I didn't see it anywhere, but may have overlooked.)